en/ros/external_position_estimation.md

```diff
@@ -1,114 +1,220 @@
+
+> **Note** If targeting `px4_fmu-v2` hardware you will also need to use a firmware version that includes the LPE module (firmware for other FMU-series hardware includes both LPE and and EKF).
+  The LPE version can be found in the zip file for each PX4 release or it can be built from source using the build command `make px4_fmu-v2_lpe`.
+  See [Building the Code](../setup/building_px4.md) for more details.
+
+#### Enabling External Pose Input
+
+The following parameters must be set to use external position information with LPE (these can be set in *QGroundControl* > **Vehicle Setup > Parameters > Local Position Estimator**).
+
+Parameter | Setting for External Position Estimation
+--- | ---
+[LPE_FUSION](../advanced/parameter_reference.md#LPE_FUSION) | Vision integration is enabled if *fuse vision position* is checked (it is enabled by default).
+[ATT_EXT_HDG_M](../advanced/parameter_reference.md#ATT_EXT_HDG_M) | Set to 1 or 2 to enable external heading integration. Setting it to 1 will cause vision to be used, while 2 enables MoCap heading use. 
+ 
+
+#### Disabling Barometer Fusion
+
+If a highly accurate altitude is already available from VIO or MoCap information, it may be useful to disable the baro correction in LPE to reduce drift on the Z axis.
+
+This can be done by in *QGroundControl* by unchecking the *fuse baro* option in the [LPE_FUSION](../advanced/parameter_reference.md#LPE_FUSION) parameter.
 
 #### Tuning Noise Parameters
 
-If your vision or mocap data is highly accurate, and you just want the estimator to track it tightly, you should reduce the standard deviation parameters, `LPE_VIS_XY` and `LPE_VIS_Z` (for vision) or `LPE_VIC_P` (for motion capture). Reducing them will cause the estimator to trust the incoming pose estimate more. You may need to set them lower than the allowed minimum and force-save. 
+If your vision or MoCap data is highly accurate, and you just want the estimator to track it tightly, you should reduce the standard deviation parameters: [LPE_VIS_XY](../advanced/parameter_reference.md#LPE_VIS_XY) and [LPE_VIS_Z](../advanced/parameter_reference.md#LPE_VIS_Z) (for VIO) or [LPE_VIC_P](../advanced/parameter_reference.md#LPE_VIC_P) (for MoCap). 
+Reducing them will cause the estimator to trust the incoming pose estimate more. 
+You may need to set them lower than the allowed minimum and force-save. 
+
+> **Tip** If performance is still poor, try increasing the [LPE_PN_V](../advanced/parameter_reference.md#LPE_PN_V) parameter. 
+  This will cause the estimator to trust measurements more during velocity estimation.
+
+  
+## Working with ROS
+
+ROS is not *required* for supplying external pose information, but is highly recommended as it already comes with good integrations with VIO and MoCap systems.
+PX4 must already have been set up as above.
+
+### Getting pose data into ROS
+
```

Comment:
@TSC21 
I copied this from the optitrack section below. 
- I suspect that each system (VIO or Mocap) will have its own way of doing this - so this is not generic?
- IF so, options are to have a few examples here, but I'm tempted remove the current content and just have a line that states that this is system specific. We can add links down to this section for Optitrack as an example. Reasonable?

Reply:
VIO and Mocap are different ways of obtaining pose data so it's not generic. Even for mocap, VICON vs Optitrack have their own setup and topics. So yes, it's reasonable to leave this as system specific.